.github/workflows/linting.yml

```diff
@@ -0,0 +1,33 @@
+name: PHP Linting (WordPress Coding Standards, PHPCS/PHPCBF)
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  phpcs-phpcbf:
+    name: Run PHPCS and PHPCBF
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 'latest'
+          tools: composer
+
+      - name: Install dependencies
+        run: composer install --no-scripts --no-progress --no-suggest
+
+      - name: Run PHPCS
+        run: composer phpcs
+
+      - name: Run PHPCBF
```

Comment:
Looking at the failing run for the previous commit, it looks like when code sniffer fails the code beautifier and fixer is not running. I can't see anything to suggest that this is just because there was nothing that could have been automatically fixed but I might be missing something.
https://github.com/ThreeRingsCIC/portal-wordpress-plugins/actions/runs/14652947709/job/41122691925